### PR TITLE
GITHUB: Follow github advises

### PR DESCRIPTION
**Pull request by anaelChardan**
[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md) 

**Description (for Contributor and Core Developer)**

[//]: <> (What does this Pull Request do? reference the related issue?)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | :negative_squared_cross_mark:
| Added Behats                      | :negative_squared_cross_mark:
| Changelog updated                 | :negative_squared_cross_mark:
| Review and 2 GTM                  | :clock1:
| Micro Demo to the PO (Story only) | :negative_squared_cross_mark:
| Migration script                  | :negative_squared_cross_mark:
| Tech Doc                          | :negative_squared_cross_mark:

:white_check_mark: Done and pass

:red_circle: Done but fail

:clock1: Done but pending

:negative_squared_cross_mark: Not needed

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<3 Thanks for taking the time to contribute! You're awesome! <3
```
Comment (a2xchip) [inline .github/PULL_REQUEST_TEMPLATE.md]: Such strings should be comments, many developers forget to delete it.

image

**Comment by a2xchip** [comment]
:smiley: Even in this PR, help text is not deleted! that is right what i am talking about :)

**Comment by anaelChardan** [comment]
Hi @a2xchip, thanks for the review, and you're right, I'll fix that :).

**Comment by a2xchip** [comment]
@anaelChardan :tada: Merged! Congrats :+1:

**Pull request merged**